recipes/smif/build.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Since you're using the `script` tag in the `meta.yaml`. you can delete the `build.bat` and `build.sh` files.

Reply:
Thanks, I'll delete them.